.ci/update-specs.yml

```
@@ -0,0 +1,57 @@
+---
+name: update specs
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: ecs-logging-go-logrus
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  ecs-logging-go-logrus:
```

Is it just a description? Can we change it to `ecs-logging`?

---

It's just a description. I'll use this project name, since it shoudl not be pointing to another one. my bad

```suggestion
  ecs-logging-php:
```